### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Sites-Edit-Web Application_Keyboard: Focus loss is observed after search or filter field.

### ISSUE THREAD

**Issue by accessibility18** [Microsoft/IIS.WebManager#172]
**Environment Details:**
URL: https://manage.iis.net/
Browser: Microsoft Edge 42.17134.1.0, Google Chrome Version 68.0.3440.106

**MAS ID Violated**: MAS2.4.3

**Repro Steps:**
1. Download and install "Microsoft IIS Administration" application.
2. Enable Internet information Services.
3. Open "Microsoft IIS" application using the URL: https://manage.iis.net/
4. Tab till "Web Sites" in the left menu section and Press Enter.
5. Tab till "More" commands button of the existing website inside the grid and Press Enter.
6. Tab till "Edit" in the expanded menu and Press Enter.
7. Tab till "Web Applications" in the left menu section and Press Enter.
8. Tab till "Create Web Application" button and Press Enter.
9. Navigate to Settings page tab.
10. Verify that there is no focus loss on the page and focus order is proper throughout the page.

**Actual Result:**
After Search or filter field focus, when user press Tab, focus is getting lost, and hence violating the proper focus order on the page.

**Expected Result:**
There should not be any focus loss on the page. Focus should move to all the interactive controls on the page properly and at any point of time, user should be able to identify the keyboard focus.

**Note**: Same issue is observed for the similar UI at different section of the application.

**User Impact:**
People with visual impairments can become disoriented when tabbing takes focus someplace unexpected or when they cannot easily find the content surrounding an interactive element.
People with mobility impairments who must rely on keyboard access for operating a page benefit from a logical, usable focus order.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0de7fbe1-ad7e-48e5-bcbb-8d986691e2b9} 

**Attachment:**
[Bug 172_Focus loss issue.pptx](https://github.com/Microsoft/IIS.WebManager/files/2332032/Bug.172_Focus.loss.issue.pptx)

**Comment by jhkimnew** [comment]
Fixed

**Comment by mstechie** [comment]
GitHubTags:#A11y_Manage.IIS.Net;
